agent/lib_aws_sdk_php.h

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright 2024 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Functions relating to instrumenting AWS-SDK-PHP.
+ */
+#ifndef LIB_AWS_SDK_PHP_HDR
+#define LIB_AWS_SDK_PHP_HDR
+
+extern void nr_aws_sdk_php_enable();
+extern void lib_aws_sdk_php_handle_version();
+extern void lib_aws_sdk_php_add_supportability_metric(const char* metric_name);
```

Comment:
Since C doesn't have the concept of namespaces, in order to avoid potential symbol symbol clashes, it is good practice to include 'namespace' as a prefix in all external functions. Therefore these two functions should be prefixed with `nr_` - a conventional namespace for all global symbols in newrelic-php-agent:
```suggestion
extern void nr_lib_aws_sdk_php_handle_version();
extern void nr_lib_aws_sdk_php_add_supportability_metric(const char* metric_name);
```

Reply:
Please see: [7d98b70](https://github.com/newrelic/newrelic-php-agent/pull/932/commits/7d98b70ca2e9ada5dc79739fefa63e535ae6220d)